force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,54 @@
+trigger AccountTrigger on Account (before insert, after insert) {
+/*
+    * Question 1
+    * Account Trigger
+    * When an account is inserted change the account type to 'Prospect' if there is no value in the type field.
+    * Trigger should only fire on insert.
+    */
+     /*
+    * Question 2
+    * Account Trigger
+    * When an account is inserted copy the shipping address to the billing address.
+    * BONUS: Check if the shipping fields are empty before copying.
+    * Trigger should only fire on insert.
+    */
+    /*
+    * Question 3
+    * Account Trigger
+	* When an account is inserted set the rating to 'Hot' if the Phone, Website, and Fax ALL have a value.
+    * Trigger should only fire on insert.
+    */
+    if(Trigger.isBefore && Trigger.isInsert){
+        for(Account acc : Trigger.new){
+            if(acc.Type == null){
+                acc.Type = 'Prospect';
+            }
+            if(acc.ShippingStreet != null){
+                acc.BillingStreet = acc.ShippingStreet;
+                acc.BillingCity = acc.ShippingCity;
+                acc.BillingState = acc.ShippingState;
+                acc.BillingPostalCode = acc.ShippingPostalCode;
+                acc.BillingCountry = acc.ShippingCountry;
+            }
+            if(acc.Phone != null && acc.Website != null && acc.Fax != null){
```

Comment:
What happens if the values are blank or empty space. Try using a string method to check more values